gunicorn/workers/sync.py

```diff
@@ -188,7 +187,7 @@ def handle_request(self, listener, req, client, addr):
                     respiter.close()
         except EnvironmentError:
             # pass to next try-except level
-            six.reraise(*sys.exc_info())
+            raise sys.exc_info()
```

Comment:
`sys.exc_info()` returns a tuple. I think using a plain `raise` would be enough here. Also, I'm not sure we still need the comment `# pass to next try-except level` though. @benoitc what do you think?